src/main/scala/play/api/cache/redis/impl/Builders.scala

```diff
@@ -37,6 +37,8 @@ object Builders {
   /** converts the future into the value */
   object SynchronousBuilder extends ResultBuilder[ SynchronousResult ] {
 
+    // INFO Await in reality does fiber fork and can result kernel panic if used
+    // recklessly
```

Comment:
Thanks for your note, I wasn't aware of it. Can you provide some source, e.g., a book or a specification?

Is there any alternative solution to use the Future inside of the synchronous code?